clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/UbamTests.scala

```diff
@@ -323,15 +323,15 @@ trait UbamTests { self: BaseIntegrationSpec =>
     } yield {
       projectResults should have length 3
       projectResults.foldLeft(succeed) { (_, result) =>
-        result.project should be(Some(project))
+        result.project should be(Some(Symbol(project)))
       }
       sampleResults should have length 2
       sampleResults.foldLeft(succeed) { (_, result) =>
         result.sampleAlias should be(Some(samples.head))
       }
       rpIdResults should have length 1
       rpIdResults.headOption.flatMap(_.researchProjectId) should be(
-        researchProjectIds.lastOption
+        Some(Symbol(researchProjectIds.last))
```

Comment:
`researchProjectIds.lastOption.map(Symbol.apply)` is better practice (even though this is test code and we know the list won't be empty)